platforms/android/com/crittercism/plugin/CDVCrittercism.java

```diff
@@ -57,16 +57,22 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
         Context context = this.cordova.getActivity();
         packageName = context.getPackageName();
         Resources resources = context.getResources();
-        String crAppID = context.getString(resources.getIdentifier(
-                APPLICATION_ID, STRING, packageName));
-        Crittercism.initialize(context, crAppID);
+        int identifier = resources.getIdentifier(APPLICATION_ID, STRING, packageName);
```

Comment:
Yes, this code will cause double initialization. Fixing it won't be difficult, I am planning to prioritize the app ID on the xml/plist file.

Some interesting fact: to update PhoneGap plugins, each of the plugin needs to be removed and reinstalled. When a user removes or reinstalls Crittercism plugin, the app ID on the xml/plist file is also gone. Technically this code is not necessary. Thoughts?